### PR TITLE
Milestone-4

### PR DESCRIPTION
**Pull request by ignatius22**
I have completed the following task for the milestone 4 assignment.

- Create a new logic folder inside src.
- initialize two javascript files in logic folder: calculate.js and operate.js.
- Implement the calculate.js module.
- Implement the operate.js module.
- Add calculate.js to the App component.
Please take a moment to review my project. I'm looking forward to your feedback. Thanks.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @ignatius22 👋

**### You've done pretty well on this**👍

- [x] Please don't push node_modules to your repo, add it to `.gitignore` and run this command on your terminal to remove it from GitHub tracking `git rm -r --cached node_modules/`

**### STATUS: CHANGES REQUIRED**👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  const firstNumber = Big(numberOne);
+  const secondNumber = Big(numberTwo);
+  let result = 0;
+
+  if (operation === '+') {
+    result = firstNumber.plus(secondNumber);
+  } else if (operation === '-') {
+    result = firstNumber.minus(secondNumber);
+  } else if (operation === '-') {
+    result = firstNumber.minus(secondNumber);
```
Comment (mikenath223) [inline src/logic/operate.js]: - [x] Why are there two similar conditions, please remove one.

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  } else if (operation === 'x') {
+    result = firstNumber.times(secondNumber);
+  } else if (operation === '÷') {
+    result = firstNumber.div(secondNumber);
+  } else if (operation === '%') {
+    result = firstNumber.mod(secondNumber);
+  }
```
Comment (mikenath223) [inline src/logic/operate.js]: - [x] `%` means a percentage value, kindly divide the number by `100` instead.

**Review by shloch — APPROVED** [review]
## Hello @ignatius22 

Great job with the corrections. You get a thumbs up
Please take note of division by ZERO in the file `/src/logic/operate.js` ( line 15), it will generate an error. Handle this in next milestone (You can return `NaN` or `Error` or `Infinity` etc)
accomplished

Thumbs up for this milestone
Happy coding

**Pull request merged**